docs/plugins/social_login.rst

```diff
@@ -1,10 +1,105 @@
 .. vale off
 
-Social Login
-#############
+Social Login Guide
+=========================
 
- .. image:: images/twilio-sid-authtoken.png
+With Mautic's Social Login, users can easily sign in via their favorite social platforms like Twitter, Facebook, or LinkedIn. The social login feature automatically pre-fills forms with profile data and updates or creates new contacts in Mautic, streamlining the user experience.
```

Comment:
With Mautic's social Login, users can easily sign in via their favorite social platforms like Twitter, Facebook, or LinkedIn. The social login feature automatically pre-fills forms with profile data and updates or creates new contacts in Mautic, streamlining the user experience.